### DISCUSSION TITLE
looking for testing / feedback for 0.9.0 (hopefully final release before 1.0)

### DISCUSSION THREAD

**Discussion by toumorokoshi** [toumorokoshi/tome#38] [General]
@zph @Nattle

Hi! you're added to this discussion because you've shown interest or have contributed to tome in the past. Thanks for your involvement!

Over the past couple months, as time allows I've been spending time cleaning up and adding the final features to get ready for a 1.0 stable release.

There's always more features to be added, but I think 0.9.0 includes all the breaking changes I think would be valuable before tome reaches some level of behavioral stability.

Here, I would love your feedback and bug reports! If you could:

- [try out the 0.9.0 release](https://github.com/toumorokoshi/tome/releases/tag/v0.9.0)
- call out any behavioral improvements you'd like to see, especially breaking ones. 
- file bugs as you encounter them.

That would be hugely helpful! If all looks good I plan on posting tome to a few channels (Hacker News / Reddit / Twitter, perhaps a youtube video) to get some more users and feedback, and if looks good there publish the 1.0 release.

Thanks for any help! and thanks again for being involved.

**Comment by Nattle** [comment]
Thanks for the heads up, I'll update it this morning and see if any of my tools aren't working.
